ably.d.ts

```diff
@@ -391,41 +809,100 @@ declare namespace Types {
+		 */
+		constructor(options: Types.ClientOptions);
+		/**
+		 * Creates an Ably client instance
+		 *
+		 * @param key An Ably API Key
+		 */
+		constructor(key: string);
 		static Crypto: Types.Crypto;
 		static Message: Types.MessageStatic;
 		static PresenceMessage: Types.PresenceMessageStatic;
 	}
 
 	class RestCallbacks extends RestBase {
+		/**
+		 * A promisified version of the library (use this if you prefer to use Promises or async/await instead of callbacks)
+		 */
 		static Promise: typeof Types.RestPromise;
 		static Callbacks: typeof Types.RestCallbacks;
+		/**
+		 * A reference to the Auth authentication object.
+		 */
 		auth: Types.AuthCallbacks;
+		/**
+		 * A reference to the `Channel` collection instance.
+		 */
 		channels: Types.Channels<Types.ChannelCallbacks>;
-		request: <T = any>(method: string, path: string, params?: any, body?: any[] | any, headers?: any, callback?: Types.StandardCallback<Types.HttpPaginatedResponse<T>>) => void;
-		stats: (paramsOrCallback?: Types.paginatedResultCallback<Types.Stats> | any, callback?: Types.paginatedResultCallback<Types.Stats>) => void;
+		/**
+		 * Makes a REST request to a provided path. This is provided as a convenience for developers who wish to use REST API functionality that is either not documented or is not yet included in the public API, without having to handle authentication, paging, fallback hosts, MsgPack and JSON support, etc. themselves.
+		 */
+		request<T = any>(method: string, path: string, params?: any, body?: any[] | any, headers?: any, callback?: Types.StandardCallback<Types.HttpPaginatedResponse<T>>): void;
+		/**
+		 * Queries the [REST `/stats` API](https://ably.com/documentation/rest-api#stats) and retrieves your application’s usage statistics. A PaginatedResult is returned, containing an array of stats for the first page of results. PaginatedResult objects are iterable providing a means to page through historical statistics. [See an example set of raw stats returned via the REST API](https://ably.com/documentation/general/statistics).
+		 */
+		stats(params?: any, callback?: Types.paginatedResultCallback<Types.Stats>): void;
+		/**
+		 * Queries the [REST `/stats` API](https://ably.com/documentation/rest-api#stats) and retrieves your application’s usage statistics. A PaginatedResult is returned, containing an array of stats for the first page of results. PaginatedResult objects are iterable providing a means to page through historical statistics. [See an example set of raw stats returned via the REST API](https://ably.com/documentation/general/statistics).
+		 */
+		stats(callback?: Types.paginatedResultCallback<Types.Stats>): void;
+		/**
+		 * Obtains the time from the Ably service as milliseconds since epoch. (Clients that do not have access to a sufficiently well maintained time source and wish to issue Ably TokenRequests with a more accurate timestamp should use the `queryTime` ClientOption instead of this method).
```

Comment:
```suggestion
		 * Obtains the time from the Ably service as milliseconds since epoch. Clients that do not have access to a sufficiently well maintained time source and wish to issue Ably `TokenRequest`s with a more accurate timestamp should use the `queryTime` `ClientOption` instead of this method.
```